### ISSUE TITLE
Multi-GPU support? 

### ISSUE THREAD

**Issue by astrude** [Russell91/TensorBox#111]
Can we have multi-gpu support for tensorbox for reinspect model? Could you also suggest if there are any frameworks to explore for speeding up the training process for Object-detection use-case? Thanks in advance.

**Comment by Russell91** [comment]
Yes, it should be possible. Maybe you can look into the possibility of
implementing it on top of the slim model?
